### PR TITLE
The Locallibrary website (part1)

### PR DESCRIPTION
**Pull request by thanhbao922003**
express express-locallibrary --view=pug
// Go to project folder
cd express-locallibrary
// Install project dependencies
npm install

npm start

### CODE REVIEW

**Comment by thanhbao922003** [comment]
ready

**Comment by thanhbao922003** [comment]
ready

**Comment by thanhbao922003** [comment]
ready

**Comment by thanhbao922003** [comment]
ready

**Comment by huongnt-2545** [comment]
Git ignore folder `dist/, node_modules/`

**Comment by huongnt-2545** [comment]
commented

**Comment by thanhbao922003** [comment]
ready

File: index.js
Comment (huongnt-2545) [inline index.js:1]: xóa file này đi e nha

**Review by huongnt-2545 — COMMENTED** [review]
E đẩy nhầm cấu trúc project lên rồi nha
E tham khảo Project structure ở slide 15 trong link này nha: https://drive.google.com/drive/u/0/folders/1fFzjp6IAwRGC8SpzjD1fetVkNGA633Zb

**Pull request closed without merging**